superduperdb/misc/files.py

```diff
@@ -23,7 +24,12 @@ def _get_file(uri):
     return file
 
 
-def load_uris(r: dict, root: str, raises: bool = False):
+def load_uris(
+    r: dict,
+    root: str,
+    encoders: t.Dict,
+    raises: bool = False,
```

Comment:
raises not being used?

Reply:
Yes it's used if the file doesn't exist.